main.tf

```
@@ -152,63 +45,49 @@ locals {
 # EKS Fully managed nodes
 ################################################################################
 locals {
+  node_labels = merge(
+    var.node_labels
+  )
```

Cleaned up, but why is this need at all?
```suggestion
  node_labels = var.node_labels
```